.github/BECOMING_A_CORE_CONTRIBUTOR.md

```diff
@@ -38,7 +38,7 @@ Here, we describe general expectations from core contributors:
 
 ### Pull Requests (PRs)
 
-- Pull requests are the evolutionary mechanism of Lightning, so quality is extremely important. Make sure contributors adhere to the guidelines described in the [contributing section](CONTRIBUTING.md#Pull-request).
+- Pull requests are the evolutionary mechanism of Lightning, so quality is extremely important. Make sure contributors adhere to the guidelines described in the [contributing section](CONTRIBUTING.md#Pull-Request).
```

Comment:
Thought this would fix the docs failures but didn't :(

I don't know what causes them

Reply:
@carmocca For the docs failures, #11894 should temporarily fix the error (warning).